src/templates/aws/codepipeline/main.tf.twig

```diff
@@ -1,6 +1,6 @@
 resource "aws_codepipeline" "{{ name }}" {
   name     = "${var.codepipeline_name}"
-  role_arn = "${var.codepipeline_role_arn}"
+  role_arn = "arn:aws:iam::${var.account_id}:role/${var.codepipeline_role_arn}"
```

Comment:
please change `var.codepipeline_role_arn` to `var.codepipeline_role_name` and everywhere similar or dependent on this reference ;)